modules/3d-tiles/src/lib/tileset/helpers/get-frame-state.js

```diff
@@ -18,7 +18,7 @@ const cullingVolume = new CullingVolume([
 export function getFrameState(viewport, frameNumber) {
   // Traverse and and request. Update _selectedTiles so that we know what to render.
   const {cameraDirection, cameraUp, height} = viewport;
-  const {metersPerPixel} = viewport.distanceScales;
+  const {metersPerUnit} = viewport.distanceScales;
 
   const viewportCenterCartographic = [viewport.longitude, viewport.latitude, 0];
```

Comment:
Not for this PR: now that we are using `viewport` (aka deck.gl `WebMercatorViewport`) as an argument, we should reduce the API surface between the two modules.

Here we are using both `viewportCenterCartographic = [viewport.longitude, viewport.latitude, 0]` and `viewportCenterCommonSpace = viewport.center`. Ideally we'll choose one and use `projectPosition`/`unprojectPosition` to convert between them. Technically those two are not the same location (center may have z).